src/main/java/org/myrobotlab/service/HobbyServo.java

```diff
@@ -114,33 +113,22 @@ public long getLastActivityTime() {
   }
 
   @Override
-  public void setOverrideAutoDisable(Boolean val) {
-    // TODO Auto-generated method stub
-    
-  }
-
-  @Override
-  public void addIKServoEventListener(IKJointAngleListener listener) {
-    // TODO Auto-generated method stub
-    
-  }
-
-  @Override
-  public void addIKServoEventListener(IKJointAnglePublisher publisher) {
-    // TODO Auto-generated method stub
-    
-  }
-
-  @Override
+  @Deprecated /* bleh - speed is the right method name */
   public void setSpeed(Double speed) {
-    // TODO Auto-generated method stub
+    // These 2 values can be tweaked for a slightly different curve that
+    // fits the observed data.
+    double slope = 3.25;

```

Comment:
seems like this should be pulled out into a static helper method and referenced as a place holder until we can remove all references to old style speed control.

Reply:
I supposed.
HobbyServo is "net new" - so I'd prefer that it get one step closer to the  

- correct implementation
- correct method name

The next step in that sequence would be setSpeed has the correct implementation - my preference would be to remove this incorrect albiet backwards-compatible implementation behind and basically be the same as setVelocity.

I know there will be about 8 steps to get back to correct implementation with correct name .. but shouldn't we take the first step with "new" stuff ?